dts/riscv/rapidsilicon/rapidsi_virgo.dtsi

```diff
@@ -150,7 +150,7 @@
 		 spi0: spi@A0800000 {
 			 compatible = "andestech,atcspi200";
 			 reg = <0xA0800000 0xFFFF>;
-			 interrupts = <24 5>;
+			 interrupts = <24 10>;
```

Comment:
My only concern is that WDT is considered as -1 interrupt. interrupt to the RISC-V code 0-18 are internal, the WDT is -1 and then timer onward is 19 onward. this way SPI is 24 interrupt number. I want to figure out what the WDT is connected to? Is it connected to reset and is the Timer connected to 19?

Reply:
Yes, the PIT is connected to interrupt 19 as it does trigger. I have also been thinking about the same WDT -1 interrupt question and how in Zephyr this gets connected. I still have to dive into WDT to check several things.